backend/drm/drm.c

```diff
@@ -605,15 +605,15 @@ static bool wlr_drm_output_set_cursor(struct wlr_output_state *output,
 
 		// OpenGL will read the pixels out upside down,
 		// so we need to flip the image vertically
-		wlr_matrix_surface(plane->matrix, plane->width, plane->height,
+		wlr_matrix_texture(plane->matrix, plane->width, plane->height,
 			output->base->transform ^ WL_OUTPUT_TRANSFORM_FLIPPED_180);
 
 		plane->wlr_rend = wlr_gles2_renderer_init();
 		if (!plane->wlr_rend) {
 			return false;
 		}
 
-		plane->wlr_surf = wlr_render_surface_init(plane->wlr_rend);
+		plane->wlr_surf = wlr_render_texture_init(plane->wlr_rend);
```

Comment:
Grepping for `surf` should be enough, I don't think there's any other shorthand in use.